multicluster/test/e2e/service_test.go

```diff
@@ -117,10 +117,10 @@ func (data *MCTestData) testServiceExport(t *testing.T) {
 	}
 
 	// Verfiy that ACNP works fine with new Multicluster Service.
-	data.verifyMCServiceACNP(t, clientPodName, westIP)
+	data.verifyMCServiceACNP(t, clientPodName, eastIP)
```

Comment:
this change is not mentioned in the PR, what's it for?

Reply:
The summary is updated, the original one is actually incorrect but the result is expected because both ways caused packet drops.